contracts/libraries/PositionValue.sol

```diff
@@ -23,23 +24,13 @@ library PositionValue {
         uint256 tokenId,
         uint160 sqrtRatioX96
     ) internal view returns (uint256 amount0, uint256 amount1) {
-        (PrincipalParams memory principalParams, FeeParams memory feeParams) =
-            _constructParams(positionManager, tokenId, sqrtRatioX96);
+        (amount0, amount1) = principal(positionManager, tokenId, sqrtRatioX96);
```

Comment:
readability nit: principal0, principal1 and fees0, fees1, then just `return (principal0 + fees0, principal + fees1)`